cosmos/config.py

```diff
@@ -42,6 +42,7 @@ class RenderConfig:
     :param dbt_deps: Configure to run dbt deps when using dbt ls for dag parsing
     :param node_converters: a dictionary mapping a ``DbtResourceType`` into a callable. Users can control how to render dbt nodes in Airflow. Only supported when using ``load_method=LoadMode.DBT_MANIFEST`` or ``LoadMode.DBT_LS``.
     :param dbt_executable_path: The path to the dbt executable for dag generation. Defaults to dbt if available on the path.
+    :param env_vars: A dictionary of environment variables for rendering. Only supported when using ``LoadMode.DBT_LS``.
```

Comment:
@jbandoro could you add this to our docs, please?

https://github.com/astronomer/astronomer-cosmos/blob/main/docs/configuration/render-config.rst

https://astronomer.github.io/astronomer-cosmos/configuration/render-config.html